### PR TITLE
Add monitoring image to workflow options

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose

- No issue is linked to this PR.

### Changes

- No changes.

### Review Instructions

- No instructions.

### CODE REVIEW

File: lira/lira_utils.py [hunk 1]
```diff
@@ -266,14 +266,17 @@ def compose_config_options(cromwell_options_file, lira_config):
         cromwell_options_file = cromwell_options_file.decode()
     options_json = json.loads(cromwell_options_file)
 
+    # If a monitoring image is defined in the config, add it to the options JSON
+    if 'monitoring_image' in lira_config.__dict__.keys():
```
Comment (tbl3rd) [inline lira/lira_utils.py]: __dict__ because lira_config is an object?

**Review by tbl3rd — APPROVED** [review]

File: lira/lira_utils.py (hunk 1 above)
Comment (samanehsan) [reply lira/lira_utils.py]: That's right!

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
I ended up changing the logic slightly...want to take another look @tbl3rd?

**Review by tbl3rd — APPROVED** [review]

**Pull request merged**